src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -59,7 +59,7 @@ static constexpr std::string_view AntialiasingModeKey{ "antialiasingMode" };
 static constexpr std::string_view TabColorKey{ "tabColor" };
 static constexpr std::string_view BellStyleKey{ "bellStyle" };
 
-static const winrt::hstring DesktopWallpaperEnum{ L"DesktopWallpaper" };
+static const winrt::hstring DesktopWallpaperEnum{ L"desktopWallpaper" };

```

Comment:
```suggestion
static const std::string_view DesktopWallpaperEnum{ L"desktopWallpaper" };
```

I think that's all you might have to do since `to_hstring` is called on this when it's needed below.

Reply:
uh this will definitely not work lol

why is it even a wide string

why did we put L on it

Reply:
Oh yeah, remove the L after making it a string_view

Reply:
Alright I think I fixed it

Reply:
Ok. I'm a dummy. Make it a `std::wstring_view` (and add the 'L' back in).

Since we're specifically comparing this to an hstring and hstrings are wide, keep this as a wstring and remove to `to_hstring` call when it's used.

Mind fixing it? It'll be a quick fix.

Reply:
Okay how about now?